content/03.ml-concepts.md

```diff
@@ -22,6 +25,9 @@ Also, many conventional metrics for classification (e.g. area under the receiver
 Model performance should be evaluated with a carefully picked panel of relevant metrics that make minimal assumptions about the composition of the testing data [@doi:10.1021/acs.molpharmaceut.7b00578], with particular consideration given to metrics that are most directly applicable to the task at hand.
 
 Extreme cases warrant testing the robustness of the model and metrics on simulated data for which the ground truth is known.
-Said simulations can be used to verify the correctness of the model’s implementation as well.
+To accurately test the performance of the model, it is important that simulated datasets be generated for a range of parameters.
+Simulated datasets generated from parameters that are consistent with the assumptions of the deep learning models can be used to verify the correctness of the model’s implementation.
+Further varying the parameters to test variation and violations of model assumptions can further test the sensitivity of performance to those assumptions. 
+Generative models based upon sample training sets can provide important tools to generate large cohorts of simulated datasets that are representative of the real world problems to which the machine learning prediction models will be applied.
```

Comment:
This is a good paragraph, just not 100% sure it belongs here. For context, in #218, I proposed removing the paragraph altogether. In response, @cgreene created #220, writing

> I am not always a big fan of data simulation, but it probably merits a mention. It was in tip 1 (removed in #218) which appears to be the wrong place. [Tip 4] might be a better spot.

I propose closing #218 and moving this paragraph to Tip 4. Does that work for you @ejfertig and @cgreene?

Reply:
I agree with the proposed change of putting data simulation into tip 4. What do you think about merging it here first and then moving/integrating into tip 4 as a second step of edits?

Reply:
Concur @cgreene

Reply:
Ok - I am going to mark approve which will enable @Benjamin-Lee to merge this if desired and then open up the PR for the second set. I'm in meetings all day today but can try to get to it tomorrow if nobody else ✅ s it first.

Reply:
whoops - I had already marked approve 😂  - i guess someone else has to 😁

Reply:
I'll merge this and do the PR for the move to tip 4. Thanks!